web-api/src/app.js

```diff
@@ -745,8 +745,8 @@ const { validatePdfLambda } = require('./documents/validatePdfLambda');
  * case-notes
  */
 {
-  app.get(
-    '/case-notes/batch-cases/:docketNumbers/user-notes',
+  app.post(
```

Comment:
Just verifying that the `get` had to change to a `post` because we needed a different way to pass the `Docket Numbers` through? What was the issue with passing them as query params? (This is mostly to satiate my curiosity :D )

Reply:
Normally, this probably wouldn't have been a big deal, but on trial sessions that can have a lot of trials (say in this instance 125 or more) caused a 414 error to be thrown.  So, to make sure the URI did not exceed the limit of the gateway, it was easier to pass in the body vs. query params.

Reply:
@mrinsin My understanding of the way [API Gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/limits.html) works, our URL is artificially limited in length. It could be that I read that documentation wrong and query params have a different effect on API Gateway than long paths do.

Alternatively we could leave the request as a GET instead of changing to POST and still include a request body. I don't particularly have any issues with that here since this API isn't exactly high on the Richardson maturity model of REST.

Reply:
Ahh thank y'all for the explanation! @jamesobrooks no worries, I'm not feeling too strongly about changing the request especially now that y'all helped me get a deeper understanding of the reason behind the change! 

Side question - can we really do a GET request with a body? I'd love to learn how if so - I feel like I tried it before and couldn't actually access the body until I changed it to a POST!

Reply:
Well, I don't know if the libraries we're using would support it out of the box.

Reply:
Awesome, thanks for knowledge sharing, y'all!

Reply:
Thanks @halprin.  I also thought you could only get access to the body using POST.

Reply:
Thanks @halprin for a much better answer than mine.